waypoints.proto

```diff
@@ -0,0 +1,194 @@
+syntax = "proto3";
+
+option java_package = "com.geeksville.mesh";
+option java_outer_classname = "WaypointType";
+option optimize_for = LITE_RUNTIME;
+option go_package = "github.com/meshtastic/go/generated";
+
+/*
+ * Goal of this type is to provide general categories for Waypoints.
+ * This way an external interface can show them as a special icons.
+ * Therefore maps with waypoints will be a lot easier to understand.
+ * Focus for categories is on navigation, organisation of equipment and special places to know about.
+ *
+ * GENERAL-POINTS: 1  - 3
+ * LAND-MARKS:     4  - 14
+ * VEHICLES:       15 - 21
+ * RESSOURCES:     22 - 31
+ */
+enum WaypointType {
+
+  /* ----- GENERAL-POINTS ----- */
+
+  /*
+   * Generic waypoint
+   */
+  GENERIC_WAYPOINT = 0;
+
+  /*
+   * Place to rally / meet
+   */
+  RALLY_POINT = 1;
+
+  /*
+   * Point with great view or to look at
+   */
+  VIEW_POINT = 2;
+
+  /*
+   * Point with dangers, warnings
+   */
+  DANGER = 3;
+
+
+
+  /* ----- LAND-MARKS ----- */
+
+  /*
+   * High point, mountain, hill, ...
+   */
+  MOUNTAIN = 4;
+
+  /*
+   * River, stream, ...
+   */
+  RIVER = 5;
+
+  /*
+   * Lake, sea, pond,  ...
+   */
+  LAKE = 6;
+
+  /*
+   * Lots of trees
+   */
+  FOREST = 7;
+
+  /*
+   * Field
+   */
+  FIELD = 8;
+
+  /*
+   * Valley, Canyon
+   */
+  VALLEY = 9;
+
+  /*
+   * City, town, village, ...
+   */
+  CITY = 10;
+
+  /*
+   * Rails for trains, street, bridge ...
+   */
+  TRANSPORT_INFRASTRUCTURE = 11;
+
+  /*
+   * General Building
+   */
+  BUILDING = 12;
+
+  /*
+   * Reserved for later use, feel free to find a meaningfull addition
+   */
+  RESERVED = 13;
+
+  /*
+   * Reserved for later use, feel free to find a meaningfull addition
+   */
+  RESERVED_2 = 14;
+
+
+
+  /* ----- VEHICLES ----- */
+
+  /*
+   * Bike human or motor powered
+   */
+  BIKE = 15;
+
+  /*
+   * Car, Pickup, Bus, Truck ...
+   */
+  CAR = 16;
+
+  /*
+   * Sailboat, Motorboat, Kanu, vehicles in water
+   */
+  SHIP = 17;
+
+  /*
+   * Train, handcar, vehicles on rails
+   */
+  TRAIN = 18;
+
+  /*
+   * Fixed wing aircrafts
+   */
+  PLANE = 19;
+
+  /*
+   * Rotary wing aircrafts
+   */
+  HELICOPTER = 20;
+
+  /*
+   * Military vehicles with armour
+   */
+  TANK = 21;
+
+
+
+  /* ----- RESSOURCES ----- */
+
+  /*
+   * A thing for something else
+   */
+  GENERIC_RESSOURCE = 22;
+
+  /*
+   * Somebody
+   */
+  PERSON = 23;
```

Comment:
Seems redundant when we have node positions already

Reply:
I thought more of a third party person. Doctor etc.